checker/jdk/nullness/src/javax/script/Bindings.java

```diff
@@ -0,0 +1,130 @@
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package javax.script;
+import java.util.Map;
+
+import org.checkerframework.checker.nullness.qual.Nullable ;
+
+/**
+ * A mapping of key/value pairs, all of whose keys are
+ * <code>Strings</code>.
+ *
+ * @author Mike Grogan
+ * @since 1.6
+ */
+public interface Bindings extends Map<String, Object> {
+    /**
+     * Set a named value.
+     *
+     * @param name The name associated with the value.
+     * @param value The value associated with the name.
+     *
+     * @return The value previously associated with the given name.
+     * Returns null if no value was previously associated with the name.
+     *
+     * @throws NullPointerException if the name is null.
+     * @throws IllegalArgumentException if the name is empty String.
+     */
+    public @Nullable Object put(String name, Object value);
+
+    /**
+     * Adds all the mappings in a given <code>Map</code> to this <code>Bindings</code>.
+     * @param toMerge The <code>Map</code> to merge with this one.
+     *
+     * @throws NullPointerException
+     *         if toMerge map is null or if some key in the map is null.
+     * @throws IllegalArgumentException
+     *         if some key in the map is an empty String.
+     */
+    public void putAll(Map<? extends String, ? extends Object> toMerge);
+
+    /**
+     * Returns <tt>true</tt> if this map contains a mapping for the specified
+     * key.  More formally, returns <tt>true</tt> if and only if
+     * this map contains a mapping for a key <tt>k</tt> such that
+     * <tt>(key==null ? k==null : key.equals(k))</tt>.  (There can be
+     * at most one such mapping.)
+     *
+     * @param key key whose presence in this map is to be tested.
+     * @return <tt>true</tt> if this map contains a mapping for the specified
+     *         key.
+     *
+     * @throws NullPointerException if key is null
+     * @throws ClassCastException if key is not String
+     * @throws IllegalArgumentException if key is empty String
+     */
+    @SuppressWarnings("nullness")    // required: @Initialized @Nullable Object, as correctly pointed out by nullness-checker is not desired as null value of key will cause NullPointerException. Here, the subclass doesn't respect annotations from superclass. 
```

Comment:
Don't mention `@initialized` if it is not relevant.
Don't mention "as ponited out by nullness-checker".  That's a distraction.  Just exlpain what the checker reports and why the code is correct.
(Likewise for two other comments in this file.)